react/features/base/util/randomUtil.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * String consisting of alphanumeric characters
+ * @const
+ */
+const ALPHANUM
+= '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+/**
+ * String consisting of hexadecimal digits.
+ * @const
+ */
+const HEX_DIGITS = '0123456789abcdef';
+
+/**
+ * Utility singleton responsible for generation of random sequences
+ *
+ * @singleton
+ * @const
+ */
+const RandomUtil = {
```

Comment:
RandomUtil is a collection of functions without state. Consequently, I see no strong reason for RandomUtil to exist in ES6 source code and be exported as such. Please remove RandomUtil and export each of its functions separately.